---
name: "Lambda Moses (aka Dongyi/Lambda Lu)"
position: "Postdoctoral Scholar"
address: "Department of Statistics,\\tabularnewline Columbia University"
pronouns: she/her
phone: +1 424 666 5899
email: "dl3764@columbia.edu"
github: lambdamoses
date: "`r format(Sys.time(), '%B %Y')`"
output: vitae::hyndman
aboutme: "ORCID: 0000-0002-7092-9427"
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)
library(vitae)
library(scholar)
```

# Research experiences
```{r}
library(tidyverse)
tribble(
    ~ What, ~ Year, ~ Institution, ~ Where, ~ Info,
    "Postdoctoral scholar", "2024-present", "Columbia University", "New York, NY",
    c("Implementing on-disk functionalities of SpatialFeatureExperiment for interoperable analyses of large spatial -omics data",
      "Finding effects of technology and spatial aggregation scale on common downstream spatial -omics analyses",
      "Using Moran eigenvector spatial filtering to identify global and local negative spatial autocorrelation in spatial -omics data",
      "PI: Bianca Dumitrascu"),
    "Postdoctoral scholar", "2023-2024", "California Institute of Technology", "Pasadena, CA",
    c("Implemented functionalities to work with large images and transcript spot geometries in SpatialFeatureExperiment",
      "PI: Lior Pachter"),
    "Graduate researcher", "2017-2023", "California Institute of Technology", "Pasadena, CA",
    c("Built Voyager geospatial exploratory spatial data analysis (ESDA) framework for spatial -omics",
      "Integrated geospatial data structure into SingleCellExperiment as a new data structure SpatialFeatureExperiment",
    "Analyses of history and sociology of the field of spatial transcriptomics in Museum of Spatial Transcriptomics book and review paper",
    "Contributed to the kallisto bustools single cell RNA-seq pseudoalignment framework",
    "PI: Lior Pachter"),
    "Undergraduate researcher", "2015-2017", "University of California, Los Angeles", "Los Angeles, CA",
    c("Genome wide association study of liver lipid phenotypes in iron overload in hybrid mouse diversity panel",
    "PI: Aldons J. Lusis")
) %>% 
  detailed_entries(What, Year, Institution, Where, Info)
```

# Education

```{r}
df <- tribble(
  ~ Degree, ~ Year, ~ Institution, ~ Where,
  "PhD in biology", "2017-2023", "California Institute of Technology", "Pasadena, CA",
  "Bachelor of Science", "2013-2017", "University of California, Los Angeles", "Los Angeles, CA",
)
df$Info <- I(list(NULL, c("Molecular, cell, and developmental biology (highest departmental honors)",
    "Computational and systems biology (valedictorian)",
    "Summa cum laude, Phi Beta Kappa")))
detailed_entries(df, Degree, Year, Institution, Where, Info)
```

# Software
Bioconductor R packages:
```{r}
library(dplyr)

bibliography_entries("packages.bib") %>%
  arrange(desc(issued))
```

# Presentations

```{r}
tribble(
    ~Year, ~What, ~Desc,
    2024, "Exploratory spatial data analysis from single molecules to multiple samples",
    "Workshop, Bioconductor Conference, Van Andel Institute, Grand Rapids, MI",
    2023, "Voyager: Exploratory spatial data analysis from geospatial to spatial -omics",
    "Workshop, Bioconductor Conference, Dana-Farber Cancer Institute, Harvard Medical School",
    2023, "Voyager: Exploratory spatial data analysis from geospatial to spatial -omics",
    "Invited talk, Cedars-Sinai Medical Center, Los Angeles",
    2023, "From geospatial to spatial -omics with SpatialFeatureExperiment and Voyager",
    "Invited talk, Advanced Biomedical Computation series, Brigham & Women's Hospital, Harvard Medical School",
    2022, "SpatialFeatureExperiment: An S4 Class Bringing Geospatial Tools To Spatial Omics",
    "Package demo, Bioconductor Conference, Seattle Children's Research Institute",
    2020, "Museum of Spatial Transcriptomics",
    "Short talk, Bioconductor Conference, virtual"
) |> 
    brief_entries(
        What, Year, Desc
    )
```

# Publications

```{r, eval=FALSE}
scholar::get_publications("JjotaJ0AAAAJ") |> 
     detailed_entries(
    what = title,
    when = year,
    with = author,
    where = journal,
    why = cites
  ) |> 
    mutate(why = unlist(why) |> as.integer(),
           why = case_when(why > 0 ~ paste0("Google Scholar citations: ", why),
                           TRUE ~ NA_character_))
```


```{r}
bibliography_entries("citations.bib")
```
